src/main/java/com/rultor/agents/IndexesRequests.java

```diff
@@ -39,8 +47,106 @@
  * @version $Id$
  */
 public final class IndexesRequests implements SuperAgent {
+    /**
+     * Xpath expression for retrieving log entries from a talk's XML.
+     */
+    private static final String ARCHIVE_LOG = "//archive/log";
+
+    /**
+     * Name of the log index attribute.
+     */
+    private static final String INDEX = "index";
```

Comment:
same here. what object is this? read this article about objects: http://www.yegor256.com/2014/11/20/seven-virtues-of-good-object.html is this `INDEX` a good object as the article explains?